walkthroughs/howto-k8s-ingress-gateway-enhancements/deployment.yaml.template

```diff
@@ -0,0 +1,150 @@
+apiVersion: apps/v1
```

Comment:
Can we follow the same format as other walkthroughs for the template yaml files? (i.e.,) a single manifest.yaml.template file that has all the resources that this walkthrough needs and place that file in the folder tied to appmesh crds API version (v1beta2).

Reply:
For debugging purpose I had separated them. I can merge them now